packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -804,14 +755,23 @@ sdk.server('https://eu.api.example.com/v14');`)
    * Compile the parameter (path, query, cookie, and header) schemas for an API operation into
    * usable TypeScript types.
    *
-   * @param operation
-   * @param operationId
    */
   prepareParameterTypesForOperation(operation: Operation, operationId: string) {
     const schemas = operation.getParametersAsJSONSchema({
       mergeIntoBodyAndMetadata: true,
       retainDeprecatedProperties: true,
-      transformer: schemaTransformer,
+      transformer: (s: SchemaObject) => {
```

Comment:
I don't like that I have this same function in `prepareResponseTypesForOperation` but because it's calling `this.addSchemToExport` I had trouble moving this function into a top-level `TSGenerator` function (i.e. changing this to `transformer: this.transformJSONSchema`) and not having it fail on calling `this.addSchemaToExport` with saying `this` was undefined.

I'm fine with the minor code duplication.